raster/r.li/r.li.padrange/padrange.c

```diff
@@ -315,8 +315,8 @@ int calculate(int fd, struct area_entry *ad, double *result)
 	              (((NS_DIST1 + NS_DIST2) / 2) / hd.rows);
 
 	/* get min and max patch size */
-	min = 1.0 / 0.0;	/* inf */
-	max = -1.0 / 0.0;	/* -inf */
+	min = INFINITY;	/* inf */
```

Comment:
This change is not Cmake related and should be separated out.

It would be ideal to have one "INFINITY" related PR...